Add details on how to instanciate ExchangeSDK (#34)
LedgerHQ/exchange-sdk: README.md

  ]
```

## Usage
First you need an instance of the ExchangeSDK:
```js
import { ExchangeSDK, QueryParams } from "@ledgerhq/exchange-sdk";

// The providerId has to be set with coordination with Ledger's team. It is your unique identifier when interacting with Ledger Live.

> const exchangeSDK = new ExchangeSDK(providerId);

Reviewer: ```suggestion
const exchangeSDK = new ExchangeSDK(PROVIDER_ID);
```

```

> When your LiveApp is called by Ledger Live through a deeplink, it will receive some informations. Check [QueryParams type](https://github.com/LedgerHQ/exchange-sdk/blob/main/lib/src/liveapp.ts) to have more details about it.

Author: You're right, this is a misuse of language, but for consistency, I think this how we call the [communication between Live and LiveApp](https://github.com/LedgerHQ/ledger-live/wiki/LLD:DeepLinking#discoverliveappid---live-discover-catalog) :)

Then you can call the swap method in order to start a new swap process.
```js
// Those are parameters that given through deeplink.
exchangeSDK.swap({
  quoteId,
  fromAccountId,
  toAccountId,
  fromAmount,
  feeStrategy,
  customFeeConfig,
  rate,
});
```

You can update some of them (ex: `quoteId`), if your interface offers the user to change those parameters.
Typically, the `quoteId` is an information coming from your system, so you can update its value if during your interaction with the user it has more mearning to do so.

### Using WalletAPI methods
The ExchangeSDK is simple wrapper around the [WalletAPI](https://github.com/LedgerHQ/wallet-api). However, you cannot instanciate twice the WalletAPI client inside you LiveApp.

> So if you want to use some methods provided by WalletAPI in your LiveApp, you have the choice:

Reviewer: ```suggestion
If you want to use a method(s) provided by WalletAPI in your Live App, you have two options:
```

 * use the WalletAPI client instance provided by the ExchangeSDK
 * provide a WalletAPI client instance to the ExchangeSDK

#### Using WalletAPI client instance
Once you have your ExchangeSDK instance, you can call [WalletAPI methods](https://github.com/LedgerHQ/wallet-api/tree/main/packages/client) through its `walletAPI` property:
```js
exchangeSDK.walletAPI.account.list()
```

> #### Providing WalletAPI client instance

Reviewer: ```suggestion
#### Option 2: Having a direct dependency with `wallet-api`
```
